.github/workflows/build.yml

```diff
@@ -17,6 +17,16 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - name: Check Dart Format
+        run: |
+          # Check for differences in dart format
+          if dart format --set-exit-if-changed .
```

Comment:
Can we reuse the script `ci/dartfmt_check.sh`? It had been added so early, but I'm not sure when it was removed someday from the action.

Reply:
@littleGnAl yes that's much better. for that script, where does dart get installed and what version does it use?

Reply:
nvm, it depends on the steps used before it